### DISCUSSION TITLE
Outdated tip

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#19561] [Q&A] [answered]
When you start the game, tips appear, among which is this one - scroll down to see new mods. This is no longer entirely relevant, now you need to scroll to the right.

There may be some more outdated, but I've only noticed one so far.

**Comment by peppy** [comment, marked as answer]
I'm confused. You still scroll down. You could *also* scroll to the right now that there are more columns, but I don't know this is really too important.

**Reply by Moltanicaa** [reply]
If it doesn't matter, then okay.
